pkg/infrastructure/gcp/clusterapi/ignition.go

```diff
@@ -56,14 +64,11 @@ func editIgnition(ctx context.Context, in clusterapi.IgnitionInput) (*clusterapi
 		}
 
 		computeAddress = computeAddressObj.Address
+	} else {
+		// In private clusters, the API and API-Int servers both point to the same internal load balancer
+		computeAddress = computeIntAddressObj.Address
```

Comment:
curious: I guess this is an additional safe guard for the [change here](https://github.com/openshift/installer/pull/9656/files#diff-56276d5381d618d46ec8d35d93210662c8fdd4c9bcd90fd36afbc6a59227eb0bR900-R905) since the check there is enough?

Is there anything similar we need to do for AWS?

Reply:
> curious: I guess this is an additional safe guard for the [change here](https://github.com/openshift/installer/pull/9656/files#diff-56276d5381d618d46ec8d35d93210662c8fdd4c9bcd90fd36afbc6a59227eb0bR900-R905) since the check there is enough?

EditIgnition() updates replaces the api and api-int URLs in several ignition files with the API and API-Int LB IPs. So, we need to change passed in LBs based on type of install. So, this check is required to generate the correct ignition files.

The previous check was to extract the correct LB IPs based on private/public install.

> 
> Is there anything similar we need to do for AWS?
Yes, now that I know this works and I don't need any additional changes.

Hope your questions are answered.

Reply:
Ah yes thanks! That confirms what I was thinking too :D Cool!